.expeditor/scripts/release_habitat/expeditor_destroy_build_channel.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Script that is run by Expeditor to unconditionally clean up the
+# release-pipeline scoped Builder channel that contains our release
+# candidates.
+#
+# A successful pipeline run will have promoted these artifacts into
+# the `dev` pipeline.
+#
+# We want to delete the channel whether or not the pipeline completes
+# successfully, because otherwise, our channels will build up, causing
+# API slowdowns :(
+
+set -euo pipefail
+
+source .expeditor/scripts/release_habitat/shared.sh
+
+export HAB_BLDR_URL="${temporary_hab_bldr_url}"
```

Comment:
should this be `acceptance_hab_bldr_url`?

Reply:
It actually is... I changed it because Shellcheck rightly pointed out that it could cause confusion between it and `ACCEPTANCE_HAB_BLDR_URL`.

Once we switch over to prod, though, I figure this will just go away.